test/framework/src/main/java/org/elasticsearch/test/rest/yaml/ESClientYamlSuiteTestCase.java

```diff
@@ -157,6 +157,10 @@ public void initAndResetContext() throws Exception {
         restTestExecutionContext.clear();
     }
 
+    protected Version overwriteEsVersion(Version esVersionFromApi) {
```

Comment:
Can you add some javadoc to this method since it's going into the test framework and can be used elsewhere?